src/equations/compressible_navier_stokes_2d.jl

```diff
@@ -1,15 +1,15 @@
 @doc raw"""
-    CompressibleNavierStokesDiffusion2D(gamma, Re, Pr, Ma_inf, equations,
+    CompressibleNavierStokesDiffusion2D(gamma, mu, Pr, R, equations,
                                         gradient_variables=GradientVariablesPrimitive())
 
 These equations contain the diffusion (i.e. parabolic) terms applied
 to mass, momenta, and total energy together with the advective terms from
 the [`CompressibleEulerEquations2D`](@ref).
 
 - `gamma`: adiabatic constant,
```

Comment:
I think we should be explicit here about which units we expect these quantities to be in (and not just "metric", since there are communities with weird notions of metric units like "cm" and "g").

Maybe, in addition, we could give a set of examples, e.g., "if you use air under standard conditions, you would give mu=xxx, Pr=0.72, R=yyy..."

Reply:
But wasn't this the whole point of keep things dimensional? It is up to the user to know what they intend to pass and which units their test problem and quantities need. The `convergence_test` is a toy, manufactured solution problem so, to me, any units would be meaningless. We can use the lid driven cavity setup to provide some guidance but I need to look up what the units were.

Reply:
Good point. Maybe we could just add a sentence about that, e.g., "Fluid properties such as `gamma`, `mu`, or `Pr` can be provided in any unit system as long as they are consistent with each other."

Reply:
I added a short sentence about this in the docstring.